plugin/teksi_wastewater/interlis/interlis_importer_exporter.py

```diff
@@ -341,6 +350,7 @@ def _export_to_intermediate_schema(
         export_model,
         file_name=None,
         selected_ids=None,
+        export_orientation=90.0,
```

Comment:
Here you also need to pass this parameter to `InterlisExporterToIntermediateSchema`

Reply:
`export_orientation=selected_orientation`

Reply:
Seems not to work like that:
`==================================== ERRORS ====================================
___________ ERROR collecting teksi_wastewater/tests/test_interlis.py ___________
teksi_wastewater/tests/test_interlis.py:8: in <module>
    from teksi_wastewater.interlis.interlis_importer_exporter import (
teksi_wastewater/interlis/interlis_importer_exporter.py:56: in <module>
    class InterlisImporterExporter:
teksi_wastewater/interlis/interlis_importer_exporter.py:156: in InterlisImporterExporter
    export_orientation=selected_orientation,
E   NameError: name 'selected_orientation' is not defined
=========================== short test summary info ============================
ERROR teksi_wastewater/tests/test_interlis.py - NameError: name 'selected_ori...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.73s ===============================
2`